### ISSUE TITLE
Spicetify making spicetify_data folders in other programs config folders

### ISSUE THREAD

**Issue by anbar48** [khanhas/spicetify-cli#467]
Hi, I installed Spicetify_cli on my Mac running  10.14.6 Mojave the other day. I noticed that a few of my programs quit working. Upon investigation of those errors I noticed that many of their configuration or data files have been overwritten by 'spicetify_data' folder. If I delete it and try to do something, this folder just comes right back. For example, one program I use, Secure Pipes (which is a program that lets me easily do SSH tunneling via SOCKS proxies) has broken. In it's application support folder, I see a folder called spicetify_data. In the logs for this app, I randomly see the path to the spicetify_data folder in it's application support. I've removed spicetify_cli to the best of my ability but I need to use these applications. I've tried deleting everything relating to both spicetify and the applications I need to use, yet this is still an issue and I'm baffled where this is coming from since I think I've removed everything relating to spicetify. Please help me if you know of this issue. Attached are screenshots that might show info. Thanks. 
<img width="762" alt="Screen Shot 2021-03-22 at 1 11 53 PM" src="https://user-images.githubusercontent.com/61296963/112030479-b2c62600-8b10-11eb-8800-aef103fce8c8.png">
<img width="882" alt="Screen Shot 2021-03-22 at 1 11 35 PM" src="https://user-images.githubusercontent.com/61296963/112030484-b3f75300-8b10-11eb-9fc1-c4a8c624cb5a.png">

**Comment by khanhas** [comment]
Is there a possibility that one of your program ran in terminal changed $HOME or $XDG_CONFIG_HOME environment variables?

**Comment by anbar48** [comment]
I'm not sure if either of those environments were changed. $HOME still appears to be /users/andrewmbarker (my username). I'm not sure that XDG is a thing on MacOS (from google searches), do you have any suggestions for changing the location of whatever $XDG_CONFIG_HOME is, because whatever the equivalent of that is on mac makes sense that it would be causing this because all the programs this affect use a config.ini format and in the logs they all just randomly print the line to the rogue spicetify_data folder and a nonexistant config.ini in there. Also thank you for helping me with this, I really appreciate this.

**Comment by anbar48** [comment]
I just figured out what was causing this. I'm not sure how, but spicetify had overwritten /bin/sh with the spicetify executable. I got the /bin/sh executable from another computer, and now everything works. It makes since why that folder was created, because every program that ran a helper shell script would cause spicetify to try to generate a config.ini within the Application Support or the app itself.
